### DISCUSSION TITLE
part-ii/2010-86/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#420] [Question comments]
Part II 2010 Logic and Set Theory: Paper 3, Section II, G 

https://questions.tripos.org/part-ii/2010-86/

**Comment by ArthurConmy** [comment]
<details>
<summary>Spoiler</summary>

Prove that transitive sets of rank $\alpha$ have elements of all ranks smaller than them by induction on $\alpha$.

Only $0$ and successor ordinals have finite sets of their rank.

Only ordinals $\alpha < \omega$ have finite transitive sets of their rank; $V_{\alpha}$ are all finite and transitive, and moreover transitive sets of rank $\alpha$ must have size at least $|\alpha|$ since by the previous part they contain elements of all ranks $\beta < \alpha$

</details>
